etc/config/cuda.amazon.properties

```diff
@@ -512,10 +512,12 @@ libs.cueigen.versions.334.version=3.3.4
 libs.cueigen.versions.334.path=/opt/compiler-explorer/libs/eigen/v3.3.4
 libs.cccl.name=CUDA Core Compute Libraries (CCCL)
 libs.cccl.description=Fundamental CUDA Compute libraries: Thrust, CUB, libcudacxx
-libs.cccl.versions=trunk:261:260:250:240:230:220
+libs.cccl.versions=trunk:270:261:260:250:240:230:220
 libs.cccl.url=http://www.github.com/NVIDIA/cccl
 libs.cccl.versions.trunk.version=trunk
 libs.cccl.versions.trunk.path=/opt/compiler-explorer/libs/cccl/trunk/thrust:/opt/compiler-explorer/libs/cccl/trunk/cub:/opt/compiler-explorer/libs/cccl/trunk/libcudacxx/include:/opt/compiler-explorer/libs/cccl/trunk/cudax/include
+libs.cccl.versions.270.version=2.7.0
+libs.cccl.versions.270.path=/opt/compiler-explorer/libs/cccl/2.7.0/thrust:/opt/compiler-explorer/libs/cccl/2.7.0/cub:/opt/compiler-explorer/libs/cccl/2.7.0/libcudacxx/include:/opt/compiler-explorer/libs/cccl/2.7.0/cudax/include
```

Comment:
I notice this path is wrong (and the others seem wrong too) - maybe this never worked? the paths seem to need a `v` in front:
```
$ ls /opt/compiler-explorer/libs/cccl/
trunk  v2.2.0  v2.3.0  v2.4.0  v2.5.0  v2.6.0  v2.6.1  v2.7.0
```

I can't see how this or any of the other versioned CCCLs could have ever worked? Can you test and confirm, and then maybe we should update all these paths? I don't know how to test CCCL myself.

Reply:
Yep, looks like you're right @mattgodbolt . That was probably my fault when I first added CCCL. 

`trunk` is working correctly, but all the other versions are not: https://godbolt.org/z/rPaacT117

Reply:
With CCCL in particular this will always work because CCCL comes with the cuda container already, and if it doesn't find an overridden version it will fall back to the CUDA version. I'm guessing that's what's happening here. We will fix this and run again. Thanks for noticing it!